### ISSUE TITLE
Installing on osx-64 installs in anaconda 4 root environment old versions due to dependencies

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#302]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/302) by Erik Kretschmer (Bitbucket: [geogob](https://bitbucket.org/geogob), GitHub: [geogob](https://github.com/geogob)).**

----------------------------------------

A clean install on osx-64 bit fails due to dependencies, more precisely install a deprecated version of mss.

With a clean/new install of anadaconda2:

```
#!bash

conda config --add channels conda-forge
conda config --add channels defaults
conda install mss
```

installs mss 1.2.2:

```
#!bash

$ conda list
...
mss                       1.2.2                    py27_0    conda-forge
...
```

Attempting to force the installation of 1.6.2 leads to UnsatisfiableError:

```
#!bash

$ conda install mss=1.6.2
Fetching package metadata .............
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - mss 1.6.2* -> pyqt 4.11.* -> qt >=4.8.6,<5.0
  - mss 1.6.2* -> pyqt 4.11.* -> sip >=4.16.4,<4.18
  - navigator-updater
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Hi  @geogob


The [feedstock](https://github.com/conda-forge/mss-feedstock/blob/master/recipe/meta.yaml) on conda-forge 


```
#!text

forces:
    - pyqt 4.11.*  # [osx]
    - qt 4.8.*  # [osx]
```
e.g. OSX Version on conda-forge:

* https://anaconda.org/conda-forge/qt/4.8.7/download/osx-64/qt-4.8.7-7.tar.bz2
* https://anaconda.org/conda-forge/pyqt/4.11.4/download/osx-64/pyqt-4.11.4-py36_2.tar.bz2

with these settings the CI is able to build our package.

You have installed to the root environment. This means a new / update of package has to fullfill all dependencies of already existing packages. 
The output shows a mismatch with sip. Because sip requires  >=4.16.4,<4.18 mss can't be installaed.

Please try the following:


```
#!text

$ conda create -n mssenv python=2
$ source activate mssenv
$ conda install mss
```

The environment enables to have seperated release versions of packages and only those packages which are needed for one software.

**Comment by ReimarBauer** [comment]
**Original comment by Erik Kretschmer (Bitbucket: [geogob](https://bitbucket.org/geogob), GitHub: [geogob](https://github.com/geogob)).**

----------------------------------------

Installation in a dedicated environment was successful. Some package had to be downgraded.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

I look into this on conda-forge at the time we build 1.7.0

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

This likly is fixed by ca9616c
